--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -395,7 +395,9 @@ if (WIN32)
 else ()
     if (APPLE)
     else()
-        install(FILES ${output_sos_${build_type}} DESTINATION "${CMAKE_INSTALL_PREFIX}")
+        if (output_sos_${build_type})
+            install(FILES ${output_sos_${build_type}} DESTINATION "${CMAKE_INSTALL_PREFIX}")
+        endif()
     endif()
     install(TARGETS OrcaSlicer RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}" BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif ()
