--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -946,7 +946,11 @@
 set(_bundled_python_abi "312")
 string(REGEX REPLACE "^([0-9]+\\.[0-9]+)\\..*$" "\\1" _bundled_python_version_short "${_bundled_python_version}")
 set(_bundled_python_root "${CMAKE_PREFIX_PATH}/libpython")
-set(Python3_ROOT_DIR "${_bundled_python_root}" CACHE PATH "Root directory for bundled Python" FORCE)
+if(EXISTS "${_bundled_python_root}/bin/python${_bundled_python_version_short}")
+    set(Python3_ROOT_DIR "${_bundled_python_root}" CACHE PATH "Root directory for bundled Python" FORCE)
+else()
+    unset(Python3_ROOT_DIR CACHE)
+endif()
 set(Python3_USE_STATIC_LIBS OFF)
 set(Python3_FIND_STRATEGY LOCATION)
 set(Python3_FIND_IMPLEMENTATIONS CPython)
